### ISSUE TITLE
BUG with the HMR on 8.5.0

### ISSUE THREAD

**Issue by DieOver** [NativeScript/NativeScript#10293]
### Issue Description

I was using these following versions and the HMR works like charm

"@nativescript/android": "~8.2.2",
"@nativescript/ios": "~8.2.2",
"@nativescript/types-android": "8.2.0",
"@nativescript/types-ios": "8.2.0"
But when i use "ns migrate" and go to these following versions the HMR stop working

"@nativescript/android": "~8.5.0",
"@nativescript/ios": "~8.5.0",
"@nativescript/types-android": "^8.2.0",
"@nativescript/types-ios": "^8.2.0",
"@nativescript/webpack": "~5.0.0"
I even make a downgrade to 8.2.2 and HMR came to work again, anyone having this issue ? how can i could make the HMR work on 8.5.0 ?

### Environment

```yaml
OS: macOS 12.4
CPU: (16) x64 Intel(R) Core(TM) i7-10700K CPU @ 3.80GHz
Shell: /bin/zsh
node: 16.19.0
npm: 8.19.3
nativescript: 8.5.3

# android
java: 11.0.16.1
ndk: Not Found
apis: 28, 29, 30, 31, 32, 33
build_tools: 29.0.2, 30.0.2, 30.0.3, 31.0.0, 32.0.0, 34.0.0
system_images: 
  - android-28 | Google Play Intel x86 Atom_64
  - android-29 | Google Play Intel x86 Atom_64
  - android-30 | Google APIs Intel x86 Atom
  - android-30 | Google Play Intel x86 Atom_64
  - android-31 | Google Play Intel x86 Atom_64
  - android-32 | Google Play Intel x86_64 Atom
  - android-33 | Google Play Intel x86_64 Atom

# ios
xcode: 13.4.1/13F100
cocoapods: 1.11.3
python: 3.10.4
python3: 3.10.4
ruby: 2.6.8
platforms: 
  - DriverKit 21.4
  - iOS 15.5
  - macOS 12.3
  - tvOS 15.4
  - watchOS 8.5
```

### Dependencies

```json
"dependencies": {
  "@nativescript-community/https": "^4.0.17",
  "@nativescript/core": "*",
  "base-64": "^1.0.0",
  "nativescript-audio": "^6.2.6",
  "nativescript-ui-sidedrawer": "^10.0.2",
  "nativescript-webview-interface": "^1.4.4",
  "utf8": "^3.0.0"
},
"devDependencies": {
  "@nativescript/android": "~8.5.0",
  "@nativescript/ios": "~8.5.0",
  "@nativescript/types-android": "^8.2.0",
  "@nativescript/types-ios": "^8.2.0",
  "@nativescript/webpack": "~5.0.0",
  "@types/base-64": "^1.0.0",
  "@types/utf8": "^3.0.1"
}
```

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by DieOver** [comment]
Even with the "@nativescript/core": "^8.5.3" and updated versions, the HMR doesnt work like in the 8.2.2

    "@nativescript/android": "~8.5.0",
    "@nativescript/ios": "^8.5.1",
    "@nativescript/types-android": "^8.5.0",
    "@nativescript/types-ios": "^8.5.0",

**Comment by rigor789** [comment]
```
npm i --save-dev @nativescript/ios@next
```
Then `ns clean` and `ns run ios` again should work.

We're aiming to release a patch version of `@nativescript/ios` this week.

**Comment by rigor789** [comment]
The runtime update has been released as [v8.5.2](https://github.com/NativeScript/ios/releases/tag/v8.5.2), update, clean and re-run and HMR should work again:

```bash
npm i --save-dev @nativescript/ios@latest
ns clean
ns run ios # or ns debug ios
```

**Comment by DieOver** [comment]
Thanks @rigor789 works like CHARM... thank so much!
